service/rpc/servicer.go

```diff
@@ -188,6 +190,47 @@ func (s *Servicer) AccountBalance(ctx context.Context, request *types.AccountBal
 		return createResponse(NewAmount(goldAmt, CeloGold)), nil
 	}
 
+	if subAccount.Address[:len("ReleaseGold")] == "ReleaseGold" {
```

Comment:
what happens if the length of `subAccount.Address` is smaller than `len("ReleaseGold")`?